pkgs/games/heroic/default.nix

```diff
@@ -119,7 +100,11 @@ in stdenv.mkDerivation rec {
     changelog = "https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/releases";
     license = licenses.gpl3Only;
     maintainers = with maintainers; [ aidalgol ];
-    platforms = [ "x86_64-linux" "x86_64-darwin" "aarch64-linux" "aarch64-darwin" ];
+    # Heroic may work on nix-darwin, but it needs a dedicated maintainer for the platform.
```

Comment:
Did this regress with this update? If so fine for me. If it theoretically could still work and you just didn't test it, please keep the platform here.

Reply:
There was a change to the upstream build that puts `x64` in the path to the helper binaries, so it looks as if they are starting to hard-code assumptions about the platform.  As for ARM Linux and Darwin in general, Heroic may theoretically run, but any game you try to launch almost certainly will not, and that is the entire point of the application.